test/infrastructure/docker/README.md

```diff
@@ -4,7 +4,7 @@ CAPD is a reference implementation of an infrastructure provider for the Cluster
 
 This is one out of three components needed to run a Cluster API management cluster.
 
-For a complete overview, please refer to the documentation available [here](https://github.com/kubernetes-sigs/cluster-api-bootstrap-provider-kubeadm#cluster-api-bootstrap-provider-kubeadm) which uses CAPD as an example infrastructure provider.
+For a complete overview, please refer to the documentation available [here](https://github.com/kubernetes-sigs/cluster-api/tree/master/bootstrap/kubeadm#cluster-api-bootstrap-provider-kubeadm) which uses CAPD as an example infrastructure provider.
```

Comment:
I know this is not in scope of this PR, but I find a little bit strange than the CAPD documentation redirects the users to CAPBK "for a complete overview"
Should we change this (or remove)